package.json

```
@@ -31,7 +31,7 @@
     "version": "npm run dist && npm run docs && git add -A"
   },
   "peerDependencies": {
-    "svelte": "^3.53.1"
+    "svelte": "^3.53.1 || ^4.0.5"
```

I don't think we should leave support for Svelte 3 here. The type definitions are now using Svelte 4 types, which I don't think would be compatible. So this would only be compatible with Svelte 3 for users who aren't doing type checking. Also, there's no reason to require Svelte 4.0.5. Using any of the prior 4.x versions would be fine
```suggestion
    "svelte": "^4.0.0"
```

---

@benmccann Good point regarding removing Svelte 3 support 👍